pkgs/os-specific/linux/kernel/linux-testing-bcachefs.nix

```diff
@@ -32,6 +32,13 @@
         sha256 = diffHash;
       };
 
-      extraConfig = "BCACHEFS_FS m";
+      extraConfig = ''
```

Comment:
Tangential, but should we move this to structuredExtraConfig? That way things like the comments here won't cause rebuilds when changed.

Reply:
Agreed.